src/drawers/arcDrawer.ts

```diff
@@ -39,6 +39,9 @@ export module _Drawer {
     public draw(data: any[], drawSteps: DrawStep[], userMetadata: any, plotMetadata: Plot.PlotMetadata) {
       // HACKHACK Applying metadata should be done in base class
       var valueAccessor = (d: any, i: number) => drawSteps[0].attrToProjector["value"](d, i, userMetadata, plotMetadata);
+
+      data = data.filter(e => !!+valueAccessor(e, null));
```

Comment:
1. It filters numbers other than 0, null, undefined, NaN, unCoercible strings. 
2. If the user wants to draw a slice with size 0, that's not drawn, correct? So filtering is not semantically correct, but achieves the same result